model-optimizer/extensions/ops/slice_like_test.py

```diff
@@ -23,14 +23,18 @@
 from mo.utils.unittest.graph import build_graph
 
 nodes_attributes = {
+    'input': {'kind': 'op', 'op': 'Const', 'shape': int64_array([3, 4]), 'value': None},
```

Comment:
Why the `value` attribute for `Const` node is not specified? It is not a real case. The Const node must have some value.